Firestore/src/Query.php

```diff
@@ -202,13 +202,18 @@ public function count(array $options = [])
      * ```
      * $sum = $query->sum();
      * ```
+     *
+     * Sum of integers which exceed maxinum integer value returns a float.
+     * Sum of numbers exceeding max float value returns `INF`.
+     * Sum of data which contains `NaN` returns `NaN`.
+     *
      * @param string $field The relative path of the field to aggregate upon.
      * @param array $options [optional] {
      *     Configuration options is an array.
      *
      *     @type Timestamp $readTime Reads entities as they were at the given timestamp.
      * }
-     * @return int
+     * @return int|float
```

Comment:
I really wish to have return types of INF and NAN documented here as well.

Reply:
INF and NAN both are of the type `float`. They are not data types so it doesn't makes sense declaring them here.

Reply:
Your call!